spec/takeaway_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'takeaway'
+require 'menu'
+require 'order'
+
+describe Takeaway do
+
+  let(:order) { Order.new }
```

Comment:
We should use doubles rather than actual instances of other classes when unit testing a class- this allows us to isolate the implementations as I was mentioning in the kick-off this morning.